pdr_backend/statutil/dist.py

```diff
@@ -0,0 +1,171 @@
+from enforce_typing import enforce_types
+import numpy as np
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+from scipy import stats
+from scipy.special import ndtr
+
+
+@enforce_types
+def plot_pdf(x):
+    fig = make_subplots(rows=1, cols=1)
+    add_pdf(fig, x, 1, 1)
+    return fig
+
+@enforce_types
+def plot_pdf_cdf_nq(x):
+    fig = make_subplots(rows=3, cols=1)
+    add_pdf(fig, x, 1, 1)
+    add_cdf(fig, x, 2, 1)
+    #add_nq(fig, x, 3, 1) # FIXME un-comment
+    return fig
+
+@enforce_types
+def plot_pdf_nq(x):
+    fig = make_subplots(rows=3, cols=1)
+    add_pdf(fig, x, 1, 1)
+    #add_nq(fig, x, 3, 1) # FIXME un-comment
+    return fig
+
+
+@enforce_types
+def add_pdf(fig, x, row: int, col: int, xaxis_title:str="x-value"):
```

Comment:
I have a small objection to this naming. Generally, I am against shortening variable names but I agree in this module where everything is shortened, let's keep it this way. However, pdf makes me believe we are doing a .pdf file. It is too much of a standard naming for something entirely different. Could we find a different naming?

Reply:
Pdf, cdf and nq are well known acronyms in statistics. And if we did the expanded version it would be way too long, and harder to understanding.

Therefore, keeping as-is. Less complex.